client/boinc_cmd.cpp

```diff
@@ -188,6 +196,167 @@ void show_alerts(RPC_CLIENT &rpc) {
     }
 }
 
+////////////// --get_task_summary start ////////////////
+
+typedef vector<const char*> STR_LIST;
+
+// given: a list of lines, each consisting of n columns
+// for each column: find the longest entry.
+// Then display the lines so the columns line up.
+//
+void show_str_lists(vector<STR_LIST> lines, int ncols) {
```

Comment:
'vector' is not a trivial object, so in this case it can't be automatically moved by the compiler, and a copy will be created.
I suggest to pass lines as a constant reference:
`const vector<STR_LIST>& lines)`